updatecli/scripts/update-coredns.sh

```diff
@@ -18,46 +47,21 @@ if [ -n "$COREDNS_CHART_VERSION" ]; then
 		# if there is a new chart, reset the package version to 00 even if the images are also updated
 		yq -i ".url = \"https://github.com/coredns/helm/releases/download/coredns-${COREDNS_CHART_VERSION}/coredns-${COREDNS_CHART_VERSION}.tgz\" |
 			.packageVersion = 00" packages/rke2-coredns/package.yaml
-		CHART_UPDATED=true
-	fi
-	GOCACHE="/home/runner/.cache/go-build" GOPATH="/home/runner/go" PACKAGE="rke2-coredns" make prepare
-
-	# 2 - check if the images have been updated
-	if [ -n "$COREDNS_VERSION" ]; then
-		current_coredns_version=$(yq '.image.tag' packages/rke2-coredns/charts/values.yaml)
-		if [ "$current_coredns_version" != "$COREDNS_VERSION" ]; then
-			echo "Updating coredns image to $COREDNS_VERSION"
-			yq -i ".image.tag = \"$COREDNS_VERSION\"" packages/rke2-coredns/charts/values.yaml
-			IMAGES_UPDATED=true
-		fi
-	fi
-	if [ -n "$AUTOSCALER_VERSION" ]; then
-		current_autoscaler_version=$(yq '.autoscaler.image.tag' packages/rke2-coredns/charts/values.yaml)
-		if [ "$current_autoscaler_version" != "$AUTOSCALER_VERSION" ]; then
-			echo "Updating autoscaler image to $AUTOSCALER_VERSION"
-			yq -i ".autoscaler.image.tag = \"$AUTOSCALER_VERSION\"" packages/rke2-coredns/charts/values.yaml
-			IMAGES_UPDATED=true
-		fi
-	fi
-	if [ -n "$NODECACHE_VERSION" ]; then
-		current_nodecache_version=$(yq '.nodelocal.image.tag' packages/rke2-coredns/charts/values.yaml)
-		if [ "$current_nodecache_version" != "$NODECACHE_VERSION" ]; then
-			echo "Updating nodecache image to $NODECACHE_VERSION"
-			yq -i ".nodelocal.image.tag = \"$NODECACHE_VERSION\" |
-				  .nodelocal.initimage.tag = \"$NODECACHE_VERSION\"" packages/rke2-coredns/charts/values.yaml
-			IMAGES_UPDATED=true
-		fi
-	fi
-
-	# 3 - update the package version accordingly
-	if [ "$CHART_UPDATED" = false ] && [ "$IMAGES_UPDATED" = true  ]; then
-		package_version=$(yq '.packageVersion' packages/rke2-coredns/package.yaml)
-		new_version=$(printf "%02d" $(($package_version + 1)))
-		yq -i ".packageVersion = $new_version" packages/rke2-coredns/package.yaml
+		mkdir workdir
+		wget -P workdir/ https://github.com/coredns/helm/releases/download/coredns-${COREDNS_CHART_VERSION}/coredns-${COREDNS_CHART_VERSION}.tgz
+		tar --directory=workdir -xf workdir/coredns-${COREDNS_CHART_VERSION}.tgz coredns/values.yaml
+		mv workdir/coredns/values.yaml workdir/coredns/values_new.yaml
+		rm workdir/coredns-${COREDNS_CHART_VERSION}.tgz
+		wget -P workdir/ https://github.com/coredns/helm/releases/download/coredns-${current_coredns_chart_version}/coredns-${current_coredns_chart_version}.tgz
+		tar --directory=workdir -xf workdir/coredns-${current_coredns_chart_version}.tgz coredns/values.yaml
+		current_autoscaler_version=$(yq '.flannel.image_cni.tag' workdir/coredns/values.yaml)
+		new_autoscaler_version=$(yq '.flannel.image_cni.tag' workdir/coredns/values_new.yaml)
```

Comment:
same here

Reply:
I wrongly copied and there wasn't a new version so it passed my tests